### DISCUSSION TITLE
Add multiple versions of social icons to the icon library

### DISCUSSION THREAD

**Discussion by hein-obox** [elementor/elementor#23716] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The social media icons in the Font Awesome library only have one version of the social media icons. I would like to have multiple versions of the social media icons, e.g. I would like have a Facebook icon, without a background color. And a Facebook icon with a squared background.

### Describe the solution you'd like

The social media icons in the Font Awesome library only have one version of the social media icons. I would like to have multiple versions of the social media icons, e.g. I would like have a Facebook icon, without a background color. And a Facebook icon with a squared background.

**Comment by loucasl** [comment]
You can already do this using the shape selector on the widget, and then customize the colours in the style tab

![image](https://github.com/elementor/elementor/assets/35241778/765f2e11-1ac1-4305-b78d-03245f5223b9)

**Reply by hein-obox** [reply]
@loucasl Thank you!!!
